src/System.Private.CoreLib/shared/System/Security/CryptographicException.cs

```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
```

Comment:
This file is missing a bunch of annotations, e.g. string message can be null, the inner Exception can be null, the string insert can be null, etc.  @safern has a pending PR to annotate all of the exception types.  I believe it includes this one, in which case you should just remove it from your PR... if it doesn't, please make sure to fix up the APIs here.

Reply:
I don't think I covered this one. I started from the ones under `System` root folder. Will continue doing more today, so I'll skip these 2.

Reply:
@safern VerificationException, CryptographicException and SecurityException